MarsRealEstateFinal/app/src/main/java/com/example/android/marsrealestate/overview/OverviewViewModel.kt

```diff
@@ -75,31 +68,18 @@ class OverviewViewModel : ViewModel() {
      * @param filter the [MarsApiFilter] that is sent as part of the web server request
      */
     private fun getMarsRealEstateProperties(filter: MarsApiFilter) {
-        coroutineScope.launch {
-            // Get the Deferred object for our Retrofit request
-            var getPropertiesDeferred = MarsApi.retrofitService.getProperties(filter.value)
+        viewModelScope.launch {
+            _status.value = MarsApiStatus.LOADING
             try {
-                _status.value = MarsApiStatus.LOADING
-                // this will run on a thread managed by Retrofit
-                val listResult = getPropertiesDeferred.await()
+                _properties.value = MarsApi.retrofitService.getProperties(filter.value)
                 _status.value = MarsApiStatus.DONE
-                _properties.value = listResult
             } catch (e: Exception) {
                 _status.value = MarsApiStatus.ERROR
                 _properties.value = ArrayList()
```

Comment:
arraylistOf()?

Reply:
Why do we need to change this?

Reply:
Oh okay we can leave as-is. I wasn't sure if one way was preferred over the other, but looking it up, either is fine.